packages/datagateway-common/src/table/table.component.tsx

```diff
@@ -163,6 +163,22 @@ const VirtualizedTable = React.memo(
       disableSelectAll,
     } = props;
 
+    // Format dates to be more readable
+    // Format only if the date has a time component
+    // (to avoid formatting dates like 2020-01-01)
+    React.useEffect(() => {
```

Comment:
This only fixes the problem in tables, and not in card view

Reply:
oh, sorry, I completely forgot about card view 😅